backend-api/template.yaml

```diff
@@ -703,6 +701,8 @@ Resources:
           Projection:
             NonKeyAttributes:
               - sessionState
+              - state
```

Comment:
Just re the conversation we had at the BE catchup this morn, do we kneed both sessionState and state in here? I'm happy that you know what this answer should be so if these values are correct please dismiss

Reply:
We don't! I removed state from one place in the template but forgot about it here. Thanks!